.ci.yaml

```diff
@@ -74,7 +85,30 @@ targets:
       channel: stable
 
   ### iOS tasks ###
-  # TODO(stuartmorgan): Swap this and ios-build_all_plugins once simulator
+  # TODO(stuartmorgan): Swap this ios_platform_tests_* once simulator
+  # tests are reliable on the ARM infrastructure. See discussion at
+  # https://github.com/flutter/plugins/pull/5693#issuecomment-1126011089
+  - name: Mac_arm64 ios_build_all_plugins master
+    bringup: true
```

Comment:
If you temporarily remove `bringup: true` to force this to run in presubmit on this PR does it pass?  In the framework repo there's a check that complains if the key is missing, but I'm not sure if it'll complain in the plugins repo.

Reply:
Looks like ci.yaml validation failure prevents any LUCI tests from running?

Reply:
Seems like the stable one worked: https://cirrus-ci.com/task/5239301319950336

The master one timed out fetching? https://github.com/flutter/plugins/pull/6984/checks?check_run_id=10915780299

Reply:
Those are the existing Cirrus versions :)

Reply:
Oh right, that's what we're doing here.